section/algorithms.html

```diff
@@ -119,6 +119,50 @@ <h3>Outgoing payment and quote grant request</h3>
             <li>
                 If |authServer| [=url/scheme=] is not `https`, return failure.
             </li>
+            <li>
+                <p>Construct a {{ GrantRequest }} |grantRequest| as follows:</p>
```

Comment:
This way, ReSpec will show variable type as well.

```suggestion
                <p>Construct a {{ GrantRequest }} |grantRequest: GrantRequest| as follows:</p>
```